packages/api/src/submittable/createSubmittable.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-dupe-class-members */
+// Copyright 2017-2019 @polkadot/api authors & contributors
```

Comment:
Is this file from `polkadot` or owned by centrality?

Reply:
This is taken from plugnet. May be I need to add our copyright,

Reply:
According to the conversation in [this thread](https://centralityteam.slack.com/archives/CNFU9UTM3/p1571338509043200), it seems it has not come across to any decision yet.

Reply:
we didn't make any changes on this file, so we should keep the header untouched